docs/modules/ROOT/pages/how-to/day2ops/override-namespace-quota.adoc

```diff
@@ -0,0 +1,46 @@
+= Override Namespace Quota
+
+This guide describes how a cluster admin can override the namespace/project quota for an organization in a single {zone}.
+
+== Prerequisites
+
+* Administrator access to the {zone}
+
+== Steps
+
+=== Method 1: Create ConfigMap manually
+
+The override is active as soon as you create a config map for the organization, but only for the next namespaces to be created.
```

Comment:
```suggestion
The override is active as soon as you create a `ConfigMap` for the organization.
```

I'm not sure the second part of the sentence is required at all, it should be obvious to a cluster admin that such a change won't retroactively allow namespaces to have been created in the past (unless I'm misunderstanding what you wanted to say with the second part).